tasks/tools/gotest-scrubbed.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+### This script is used to run go test and scrub the output, the command can be used as follow:
+### ./gotest-scrubbed.sh <packages comma separated> -- <go tests flags>
+set -euo pipefail
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
# bash strict mode, fail if go test fails
set -euo pipefail
```

Reply:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Does it apply the pipe if go test fails?

Reply:
Yeah that's a valid point. But I tested and it does. 
According to bash doc it always apply all the pipes, it only change the returned error value.
> If set, the return value of a pipeline is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands in the pipeline exit successfully